### PR TITLE
Launch binary with nsjail

### PR DESCRIPTION
**Pull request by kiwiyou**
From #1. `launch` can get path for nsjail binary. Any nsjail flags that seemed useful for worker is in `LaunchOption`. Use `Path` / `PathBuf` for file paths / directories for in favor of consistency, and `Output` for process results.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1,40 +1,77 @@
...
 /// Options provided for compiling binaries.
 struct CompileOption {
     /// File to execute when compile.
-    pub compiler: OsString,
+    pub compiler: PathBuf,
     /// Arguments passed to compiler process.
     pub args: Vec<OsString>,
 }
 
-/// Result of compilation.
```
Comment (RanolP) [inline src/lib.rs:15]: Should we alter `CompileResult` with `std::process::Output`? We could add metadata like time consumed for compilation (for Compile Time Limit Exceeded), structured compile error explanation (for example, what line causes a syntax error), etc.

**Review by RanolP — COMMENTED** [review]
Except for the `CompileResult` -> `std::process::Output` transition, the code looks good to me.

File: src/lib.rs (hunk 1 above)
Comment (kiwiyou) [reply src/lib.rs:15]: We should kill compile process to limit compile time (**Will change.**), therefore time consumed in compilation need not be measured directly. Compile error is guaranteed to be provided as just a byte sequence, so it structuring could be handled at web. I don't think we can get any other meaningful information from the running proccess than `Output`.

Comment (RanolP) [reply src/lib.rs:15]: ok then it's ok to use Output directly

**Review by RanolP — APPROVED** [review]

**Pull request merged**